### PR TITLE
[Reader] Update next previous buttons

### PR DESCRIPTION
**Pull request by NickHeiner**
image

Connect #2076.

### CODE REVIEW

File: app/assets/stylesheets/_reviewer.scss [hunk 1]
```diff
@@ -191,9 +241,16 @@ svg[aria-hidden=true],
   line-height: normal;
 
   &:hover {
+    // Defined in USWDS
+    $primary-alt: #02bfe7;
```
Comment (mdbenjam) [inline app/assets/stylesheets/_reviewer.scss]: This should already be defined as $color-primary-alt

File: client/app/components/PdfUI.jsx [hunk 2]
```diff
@@ -58,12 +58,34 @@ export class PdfUI extends React.Component {
...
-        <span className="doc-list-progress-indicator">{this.props.docListIsFiltered && <FilterIcon />}
-          Document {currentDocIndex + 1} of {this.props.filteredDocIds.length}
-        </span>
+      return <div className="cf-pdf-footer cf-pdf-toolbar">
+        { this.props.prevDocId &&
+          <div className="cf-pdf-footer-buttons-left">
+            <Button
+              name="previous"
+              classNames={['cf-pdf-button']}
+              onClick={this.props.showPdf(this.props.prevDocId)}
```
Comment (mdbenjam) [inline client/app/components/PdfUI.jsx]: I know this was just moved from below, and I'm not a render police expert--but aren't we creating a new closure on every render here?

Comment (NickHeiner) [inline client/app/components/PdfUI.jsx]: Yes. This comes from `DecisionReviewer`, and I didn't want to change it in this PR. But it is a problem.

File: app/assets/stylesheets/_reviewer.scss (hunk 1 above)
Comment (NickHeiner) [inline app/assets/stylesheets/_reviewer.scss]: Good catch.

File: client/app/components/PdfUI.jsx (hunk 2 above)
Comment (mdbenjam) [inline client/app/components/PdfUI.jsx]: Cool. I feel like this:
maxresdefault

Comment (mdbenjam) [inline client/app/components/PdfUI.jsx]: Addressing it later is fine.

**Pull request merged**